docs/docs/concepts/retrieval_unstructured.mdx

```diff
@@ -0,0 +1,160 @@
+|---------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Multi-query](/docs/how_to/MultiQueryRetriever/)   | When you need to cover multiple perspectives of a question. | Rewrite the user question from multiple perspectives, retrieve documents for each rewritten question, return the unique documents for all queries. |
+| [Decomposition](https://github.com/langchain-ai/rag-from-scratch/blob/main/rag_from_scratch_5_to_9.ipynb) | When a question can be broken down into smaller subproblems. | Decompose a question into a set of subproblems / questions, which can either be solved sequentially (use the answer from first + retrieval to answer the second) or in parallel (consolidate each answer into final answer).                                                           |
+| [Step-back](https://github.com/langchain-ai/rag-from-scratch/blob/main/rag_from_scratch_5_to_9.ipynb)     | When a higher-level conceptual understanding is required. | First prompt the LLM to ask a generic step-back question about higher-level concepts or principles, and retrieve relevant facts about them. Use this grounding to help answer the user question. [Paper](https://arxiv.org/pdf/2310.06117).                                            |
+| [HyDE](https://github.com/langchain-ai/rag-from-scratch/blob/main/rag_from_scratch_5_to_9.ipynb)          | If you have challenges retrieving relevant documents using the raw user inputs. | Use an LLM to convert questions into hypothetical documents that answer the question. Use the embedded hypothetical documents to retrieve real documents with the premise that doc-doc similarity search can produce more relevant matches. [Paper](https://arxiv.org/abs/2212.10496). |
+
+:::tip
+
+See our RAG from Scratch videos for a few different specific approaches:
+- [Multi-query](https://youtu.be/JChPi0CRnDY?feature=shared)
+- [Decomposition](https://youtu.be/h0OPWlEOank?feature=shared)
+- [Step-back](https://youtu.be/xn1jEjRyJ2U?feature=shared)
+- [HyDE](https://youtu.be/SaDzIVkYqyY?feature=shared)
+
+:::
+
+## Indexing Strategies
+
+Documents needed to be indexed, frequently using embedding models to [compress the semantic information in documents to fixed-size vectors](/docs/concepts/#embedding-models).
+
+Many RAG approaches focus on splitting documents into chunks and retrieving some number based on similarity to an input question for the LLM. 
+
+But chunk size and chunk number can be difficult to set and affect results if they do not provide full context for the LLM to answer a question. 
+
+Furthermore, LLMs are increasingly capable of processing millions of tokens. 
```

Comment:
Latency is important for many applications. 

Pulling in the content of the top 10 search hits can add significant latency. 

In an extreme case, all 10 hits come from different documents.

In many domains, document length can run into the hundreds of pages (i've dozens if not hundreds of thousand page long documents in finance)

Even if the llm context window were unlimited (and could incorporate information from the context perfectly), implementation details still introduce a tradeoff in terms of latency.

Is a user willing to wait for an extra 5-60 seconds to get an answer from a chat system? This ends up being very use case dependent.